ui/tutorial-renderer/src/main/res/menu/tutorial_tips_menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/action_tips_skip"
+        android:title="@string/tutorial_tips_action_skip"
+        app:showAsAction="always" />
```

Comment:
Does this attribute change styling in some way?

Reply:
so, this is menu XML, it controls what menus items are present in the toolbar. by default the toolbar decides how many items to show directly as actions, and the rest get put into the action overflow menu. this attribute forces this action to always be shown directly and never be put in the action overflow menu

Reply:
Ah okay makes sense!